doc/Developing.md

```diff
@@ -235,33 +235,61 @@ etc.) by opening `packages/<package name>/coverage/index.html`.
 
 ## Building on Windows
 
-Run cmd.exe as an administrator and install `choco` by copy-pasting the command
-to your console:
+1. Run cmd.exe as an Administrator. You can do this by right-clicking "Command Prompt" in the Windows Start Menu, and then right-clicking "Run as Administrator".
```

Comment:
> You can do this by right-clicking "Command Prompt" in the Windows Start Menu

I must be blind, where is the "Command Prompt'?

![screen shot 2018-08-31 at 10 03 00](https://user-images.githubusercontent.com/1405703/44900647-6bcf7580-ad05-11e8-893b-16bf6e27b2e9.jpg)

Reply:
Which version of Windows do you have? Mine looks significantly different to that...

Reply:
Windows 10 Pro.

Reply:
😄 I was confused. I did the right click on the Start Menu. Great, then it is OK.
By the way there is another way without clicking: https://superuser.com/questions/968214/open-cmd-as-admin-with-windowsr-shortcut

 - <kbd>Win</kbd>+<kbd>R</kbd>
 - Type: `cmd`.
 - <kbd>Alt</kbd>+<kbd>Ctrl</kbd>+<kbd>Enter</kbd>

Reply:
There is one small mistake here in the choco command for yarn.  You should use the -i option to ignore dependencies or you will find that yarn will install Node 10+ as well, overriding your initial Node LTS install.